tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py

```diff
@@ -116,13 +116,16 @@ def setup(self):
         if (device.backend.name == 'native'
                 and not chainerx.linalg._is_lapack_available()):
             pytest.skip('LAPACK is not linked to ChainerX')
-        self.check_backward_options.update({'rtol': 5e-3})
-        self.check_double_backward_options.update({'rtol': 5e-3})
+        self.check_backward_options.update({
+            'eps': 1e-5, 'rtol': 1e-4, 'atol': 1e-4})
+        self.check_double_backward_options.update({
+            'eps': 1e-5, 'rtol': 1e-4, 'atol': 1e-4})
```

Comment:
We've been working on fixing flaky tests for other linalg routines. How about only modifying these parameters for the Cholesky tests?

Reply:
I moved these tolerance parameters to the Cholesky test.